src/stylesheets/components/_card.scss

```diff
@@ -44,13 +45,15 @@
     }
 
     &--left, &--center {
+      svg {
+        @include u-margin-right(1);
+      }
       > * {
         display: inline-block;
         vertical-align: bottom;
       }
     }
 
-
     &--stacked, &--center {
       text-align: center;
```

Comment:
Just another example of 'please, replace with USWDS utility'.  There is more but I won't make a comment for each